magento/adobe-stock-integration
# #464 : Image name popup: displaying file extension separately

### Description (*)


This PR will improve the styling of the image file name save popup and added image extension as non-editable at the end of input

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#464: Image name popup: displaying file extension separately

### Manual testing scenarios (*)

1. Login to admin panel
2. Open Magento Media Gallery (i.e. go to Cms -> Pages, edit the page and insert image)
3. Click "Search Adobe Stock" button to open images grid
4. Click on any image to expand image preview
5. Click on "Save Preview" button
6. Expected Result
![image](https://user-images.githubusercontent.com/39480008/65665444-2b9e0c00-e059-11e9-928b-1d5eb998739e.png)
